### ISSUE TITLE
SSH key and usage should be linked

### ISSUE THREAD

**Issue by wildintellect** [MAAP-Project/maap-documentation#116]
Using an SSH key to access ADE workspaces is complicated by the fact that the 2 pages about it are not cross linked to each other.
https://docs.maap-project.org/en/latest/intro/getting_started.html#Uploading-your-public-SSH-key
https://docs.maap-project.org/en/latest/platform_technical_documentation/ssh.html

**Comment by abarciauskas-bgse** [comment]
@wildintellect is this still relevant?

**Comment by wildintellect** [comment]
Yes, the question is should it be assigned to someone on the Platform team  @gchang @hookhua

**Comment by rtapella** [comment]
SSH page is here now: https://docs.maap-project.org/en/getting-started-pr-branch/system_reference_guide/ssh.html
